### PR TITLE
Fixes YAML of coding-standards.yml

### PR DESCRIPTION
**Pull request by k00ni**
Our coding standards workflow is failing for a while: https://github.com/smalot/pdfparser/actions/workflows/coding-standards.yml

Example: https://github.com/smalot/pdfparser/actions/runs/763299759

Error: `You have an error in your yaml syntax on line 33`

### CODE REVIEW

**Comment by k00ni** [comment]
PHP CS Fixer step is failing with:

```
PHP CS Fixer 2.18.6 Remote Void by Fabien Potencier and Dariusz Ruminski
Runtime: PHP 7.4.16
Loaded config default from "/home/runner/work/pdfparser/pdfparser/.php_cs".
Error: Start tag expected, '<' not found on line 1, column 1

dev-tools/vendor/bin/php-cs-fixer fix --verbose --dry-run --format=checkstyle
<?xml version="1.0" encoding="UTF-8"?>
<checkstyle/>
```

Ref: https://github.com/smalot/pdfparser/pull/413/checks?check_run_id=2436495764

@j0k3r do you have an idea?

File: .github/workflows/coding-standards.yml [hunk 1]
```diff
@@ -9,29 +9,29 @@ on:
...
+          tools: "cs2pr, composer:v1"
+      -
+        name: Install dependencies with Composer
+        uses: ramsey/composer-install@v1
+      -
+        name: Install dev tools
+        run: make install-dev-tools
+      -
+        name: Run PHP CS Fixer
+        run: 'make run-php-cs-fixer ARGS="--verbose --dry-run --format=checkstyle" | cs2pr'
```
Comment (j0k3r) [inline .github/workflows/coding-standards.yml:34]:
```suggestion
        run: 'make run-php-cs-fixer ARGS="--verbose --dry-run"'
```

**Review by j0k3r — COMMENTED** [review]
I've no idea, maybe we should remove the `cs2pr` option and just dump result in the log instead of having them in the files changed.

**Comment by k00ni** [comment]
👍🏿 All good now!

**Review by j0k3r — APPROVED** [review]
👍

**Pull request merged**